test/extended/networking/util.go

```diff
@@ -211,3 +247,7 @@ func checkConnectivityToHost(f *e2e.Framework, nodeName string, podName string,
 	defer podClient.Delete(podName, nil)
 	return waitForPodSuccessInNamespace(f.Client, podName, contName, f.Namespace.Name)
 }
+
+func isMultitenant() bool {
+	return os.Getenv("OPENSHIFT_NETWORK_PLUGIN") == "redhat/openshift-ovs-multitenant"
```

Comment:
Consider setting a separate flag to indicate whether namespaces should be isolated from a networking perspective.  There is ongoing work to make the e2e tests consumable downstream and it may make sense to allow other plugins to be able to run the isolation tests.  It also might make more sense if isolation checks are ever submitted upstream.